### DISCUSSION TITLE
Element Manager: List pages/post with element in use

### DISCUSSION THREAD

**Discussion by sflwa** [elementor/elementor#28778] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I have an add-on with an element in use one time and I have no idea which page / post it is being used on

### Describe the solution you'd like.

I'd like the ability to click on the use count for a given element and see a list of all the places it is currently being used.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by micmiller-apptio** [comment]
Echoing this request per an earlier request for the same feature.
https://github.com/orgs/elementor/discussions/24748
